sync/views.py

```diff
@@ -157,6 +160,14 @@ def transform(actblue_values):
                     timezone('America/Los_Angeles')).replace(tzinfo=None).isoformat()
         except Exception:
             continue
+
+        """
+        check for required fields
+        """
+        for fkey in knack_required_fields:
+            if fkey not in knack_lineitem:
+                logger.warning(
+                    'ActBlue data warning: {} not found'.format(fkey))
```

Comment:
Indexing on the Knack IDs will make debugging a bit of a hassle, because we'll need to look up which ActBlue field maps to the mysterious Knack number. I suppose we could make KNACK_DONOR_REQUIRED_FIELDS a dict, with the Knack ID as the key, and a pretty name as the value, then have the warning print the value. But I'm not wedded to this. We could also leave a comment that reminds Future Us how to figure out which field ActBlue seems not to be sending.